### DISCUSSION TITLE
[General] <Title>

### DISCUSSION THREAD

**Discussion by huuhoa143** [zkSync-Community-Hub/zksync-developers#265] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

How can i withdraw native token from contract. Im using this code for withdraw function
function withdraw() external onlyOwner {
        payable(owner()).transfer(address(this).balance);
    }
  When i tried to execute it, i met this error 
![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/39357066/5fb8693c-7222-4567-b1c5-204703a32b64)
And i tried to increate gas limit to execute and met another error Error function_selector = 0x, data = 0x
https://explorer.zksync.io/tx/0x8bd550bfed0bb73c07ed359a1f931bd8c412367f95ebb26c37e69774ddb63a00

![image](https://github.com/zkSync-Community-Hub/zksync-developers/assets/39357066/8cadd510-f25c-4e9e-b703-cf0bd498f275)

### What is the specific issue or error you're encountering?

I met this error when executing withdraw eth out of contract using ```transfer``` function. Here is an error message "Error function_selector = 0x, data = 0x"

### Can you share the error messages or logs you're receiving, if any?

Error function_selector = 0x, data = 0x

### Have you made any recent changes to the contract before encountering this issue?

No

### Are there any external libraries or contracts that your contract interacts with?

openzeppelin

### Can you provide the relevant portions of your contract code where the issue is occurring?

Here is contract code
https://explorer.zksync.io/address/0x80Fdc094babD77B4c3a62Ad96f388Df86E6F3A9D#contract
```
function withdraw() external onlyOwner {
        payable(owner()).transfer(address(this).balance);
    }
```

### Have you tried to isolate the problem, and if so, what were the results?

Met another erro message "Error function_selector = 0x, data = 0x"

### What steps have you already taken to try to resolve the issue?

Increase gas limit to fork execute

**Comment by vasyl-ivanchuk** [comment, marked as answer]
Hi @huuhoa143 👋 ,

This issue is a duplicate. Please check out https://github.com/zkSync-Community-Hub/zksync-developers/discussions/229#discussioncomment-7927061 for the most recent update.
